lib/wovnrb/store.rb

```diff
@@ -23,7 +23,7 @@ def self.default_settings
         'lang_param_name' => 'wovn',
         'query' => [],
         'ignore_class' => [],
-        'api_url' => 'https://wovn.global.ssl.fastly.net/v0/',
+        'api_url' => 'https://wovn.global.ssl.fastly.net/',
```

Comment:
it's better to have the url with no trailing slash, and add /v0 and /widget to it. p.s. you also need to change the usage that needs /v0